### DISCUSSION TITLE
yaml: line 27: mapping values are not allowed in this context

### DISCUSSION THREAD

**Discussion by itnetsteven** [qe201020335/BeatTogether-DockerCompose#1] [Q&A]
I have an issue when executing the docker-compose up -d like:

yaml: line 27: mapping values are not allowed in this context

What can I do, I'm using the Ubuntu 18,

**Comment by itnetsteven** [comment]
solved.
Just vi new file and copy the yml contents inside directly instead upload the file.

**Reply by qe201020335** [reply]
What do you mean by "upload the file"?
I'm guessing you did not clone the repo on your Ubuntu machine?
